migrations/20181208210402-create-users.js

```diff
@@ -9,37 +8,25 @@ export default {
       type: Sequelize.INTEGER
     },
     email: {
-      type: Sequelize.STRING,
-      allowNull: true,
+      type: Sequelize.STRING
     },
     password: {
-      type: Sequelize.STRING,
-      allowNull: true,
+      type: Sequelize.STRING
     },
     username: {
-      type: Sequelize.STRING,
-      allowNull: false,
-    },
-    emailVerification: {
-      type: Sequelize.STRING,
-      allowNull: true,
+      type: Sequelize.STRING
     },
     facebookId: {
-      type: Sequelize.STRING,
-      allowNull: true,
+      type: Sequelize.STRING
     },
     googleId: {
-      type: Sequelize.STRING,
-      allowNull: true,
+      type: Sequelize.STRING
     },
     twitterId: {
-      type: Sequelize.STRING,
-      allowNull: true,
+      type: Sequelize.STRING
     },
-    resettingPassword: {
-      type: Sequelize.STRING,
-      allowNull: false,
-      defaultValue: false
+    emailVerification: {
+      type: Sequelize.STRING
```

Comment:
Good job on the refactoring. However, shouldn't `allowNull` be set to `false` here?